packages/components/src/toolbar/toolbar-container.js

```diff
@@ -12,12 +12,17 @@ import { forwardRef } from '@wordpress/element';
  * Internal dependencies
  */
 import ToolbarContext from '../toolbar-context';
+import { getRTL } from '../utils/rtl';
```

Comment:
Ideally, we'd drop this in favor of `{ isRTL } from '@wordpress/i18n'` but this could be separate as this is already used.